src/app/components/progress-indicator/waterfall-progress-indicator/index.html

```diff
@@ -0,0 +1,325 @@
+<sky-tabbed-demo-page
+  pageTitle="Waterfall Progress Indicator"
+>
+
+  <sky-demo-page-summary>
+    <p>
+      The progress indicator module visually represents progress through sequential steps toward a goal. Waterfall progress indicators walk users through discrete steps on a page.
+    </p>
+  </sky-demo-page-summary>
+
+  <div class="sky-margin-stacked-content-section">
+    <sky-demo-page-example>
+      <sky-progress-indicator-demo>
+      </sky-progress-indicator-demo>
+    </sky-demo-page-example>
+  </div>
+
+  <sky-tabset id="content-tabs">
+    
+    <sky-tab
+      tabHeading="Design">
+    
+      <sky-demo-page-content sectionHeading="Usage">
+        <h3 class="sky-text-success">Use when</h3>
+        <sky-docs-visual-example>
+          <sky-docs-visual-example-text>
+            <p>Use waterfall progress indicators for long or complex setup processes that users typically complete only once. Waterfall indicators are ideal when events outside of the system force users to complete tasks in multiple sittings or when users must enter large amounts of data.</p>
+            <p>Use waterfall progress indicators when processes include at least three steps and the number and order of steps do not change.</p>
```

Comment:
We say the best experience requires at least three steps. Should we also recommend an upper limit for the best experience?

Reply:
Also, should this second paragraph really be treated as part of the first Use when item? It seems like it’s completely distinct from the previous paragraph to me. And it even feels like maybe it should be restated as a Don’t use when item. I think that what we are trying to say is to NOT use waterfalls when the number or order of steps changes, not TO use it whenever processes have stable steps. Same with the number of steps. It seems like maybe we want to recommend NOT to use this pattern for fewer than 3 steps, not to recommend it for process with more than 3 steps.

Reply:
Ported over to https://github.com/blackbaud/skyux2-docs/pull/625.